src/originalBudgetTrackingApp/export/outputVendors/index.test.js

```diff
@@ -1,33 +0,0 @@
-/* eslint-disable global-require, import/no-dynamic-require */
```

Comment:
This file was about test the `outputVendor` interface. We don't need this with TS